whats-new-in-pgroll-0-7-0.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: pgroll 0.7.0 update
+description: Learn about the latest changes in pgroll in the 0.7.0 release as we continue to build and turn it into a first-class open-source schema migration tool for Postgres.
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-update-0-6-0@2x.jpg
+  alt: pgroll
+author: Andrew Farries
+authorEmail: andrew@xata.io
+date: 10-17-2024
+tags: ['open-source', 'postgres', 'schema', 'migrations', 'pgroll', 'fpSchemaMigrations', 'oss']
+published: true
+slug: pgroll-0-7-0-update
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-update-0-6-0@2x.jpg
+---
+
+[pgroll](https://xata.io/pgroll) is Xata's open-source schema migration tool for Postgres. The feature that sets it apart from other migration tools is its ability to perform multi-version schema migrations using the expand/contract pattern: while a migration is in progress, `pgroll` is able to present two versions of your database schema to client applications - the old version and the new. This means that client applications that depend on the old version of the database schema continue to function while the migration is in progress, making application deployments much simpler. Gone are the days of having to perform [complicated multi-step workflows](https://planetscale.com/docs/learn/handling-table-and-column-renames#how-to-rename-a-column-on-planetscale) to do simple schema changes without breaking clients.
+
+`pgroll` is open-source under the Apache-2.0 license and developed in the open on [GitHub](https://github.com/xataio/pgroll).
+
+We released [pgroll 0.1.0](https://xata.io/blog/pgroll-schema-migrations-postgres) in October 2023. Since then we've made [nine further releases](https://github.com/xataio/pgroll/releases) as we continue to build `pgroll` and turn it into a first-class open-source schema migration tool for Postgres.
```

Comment:
```suggestion
Since the last pgroll [0.6.0 release](https://xata.io/blog/pgroll-0-6-0-update) in July 2024, we've introduced key updates aimed at solving common database migration challenges. Support for Postgres 17 ensures compatibility with the latest version, while improvements to backfill performance—now up to 80% faster—reduce downtime during large schema changes. Configurable batch sizes and delays offer developers more control over the migration process, optimizing performance and minimizing impact on their Postgres database. These enhancements make migrations faster, safer, and more efficient, especially for complex or large datasets. 
```